frontend/interfaces/schedulable_query.ts

```diff
@@ -14,6 +14,7 @@ export interface ISchedulableQuery {
   interval: number;
   platform: SelectedPlatformString; // Might more accurately be called `platforms_to_query` – comma-sepparated string of platforms to query, default all platforms if ommitted
   min_osquery_version: string;
+  last_executed?: string;
```

Comment:
What's this?

Reply:
Whoops, that's a parameter for a query on a pack. Probably going to be on a future iteration when we save live query results

Reply:
Removing!